### DISCUSSION TITLE
黴漿菌肺炎

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1741] [General]
近期大陸小孩間發展了新的肺炎感染--黴漿菌肺炎，而他與前段時間流行的武漢肺炎病毒其實是類似的，但前者的受眾群大多為小孩，而後者則是所有人群。而前者存在已久，而後者則可能是人造或新型病毒。
我認為會造成黴漿菌肺炎大量流行的原因可能是由於過去兩年大量的封鎖管理，使得人們缺少記憶型的細胞，使只要有少量細菌入侵便可產生大量反應，以上的想法僅是推測，也希望不要在台灣造成大量流行。
參考資料：https://udn.com/news/story/7266/7601683

**Comment by uwje** [comment]
人們其實很需要記憶型的免疫細胞，以應對快速的病毒入侵，因為如果沒有這個記憶型的細胞，人的免疫系統會比有記憶型細胞存在時晚大越三到五天才發揮作用，所以平時些微的病毒時有助於免疫系統的運作。

**Comment by spermwhale464** [comment]
室友一直咳，我都快被傳染了QQ
